solver/pb/caps.go

```diff
@@ -440,4 +441,10 @@ func init() {
 		Enabled: true,
 		Status:  apicaps.CapStatusExperimental,
 	})
+
+	Caps.Init(apicaps.Cap{
```

Comment:
Maybe comment this in until the follow-ups are addressed. Looks like incompatible API changes are coming for sure.